src/atoms/Button/__tests__/Button.native.test.js

```diff
@@ -1,11 +1,139 @@
 import React from 'react';
-import { renderWithTheme } from '../../../_helpers/testing';
+import { fireEvent } from '@testing-library/react-native';
 
 import Button from '../index';
+import { renderWithTheme } from '../../../_helpers/testing';
 
 describe('Renders <Button /> correctly', () => {
-  it('snapshot testing', () => {
-    const { container } = renderWithTheme(<Button onClick={() => {}}>{'Click Me'}</Button>);
+  it('Renders a Primary <Button />', () => {
+    const { container, getByText } = renderWithTheme(<Button icon="info">Click Me</Button>);
+    expect(getByText(/click me/i)).toBeTruthy();
+    expect(container).toMatchSnapshot();
+  });
+
+  it('Renders a Primary <Button /> with left icon', () => {
```

Comment:
can we follow a convention of taking one prop and writing tests for that one prop at a time and not jumble that up?
for example if you want to test iconAlign prop then add all the tests for that variations? It's simple to write for you and also simple to review for others